lib/api/graph.js

```diff
@@ -108,13 +108,69 @@ function createEmbed(graphJSON, options, callback){
     client.request("POST", "/graph/embed", params, callback);
 }
 
+/*section: graph
+ *comment: delete an embed with a specific id
+ *params:
+ *  embedId: the id of the embed to delete
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  var embedid = "foo";
+ *  dogapi.graph.deleteEmbed(embedid, function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function revokeEmbed(embedId, callback){
+    client.request("GET", "/graph/embed/" + embedId + "/revoke", callback);
+}
+
+
+/*section: graph
+ *comment: get all embeds from datadog
+ *params:
+ *  callback: function(err, res)
+ *example: |
+ *  ```javascript
+ *  dogapi.graph.getEmbeds(function(err, res){
+ *    console.dir(res);
+ *  });
+ *  ```
+ */
+function getEmbeds(callback) {
```

Comment:
Could we rename this to `getAll` to be consistent with other places?

https://github.com/brettlangdon/node-dogapi/blob/e58e06bfa8f4fde1262e87cb4970da5600f28468/lib/api/downtime.js#L165